Apps/WebClient/src/ClientApp/src/views/releaseNotes.vue

```diff
@@ -10,6 +10,30 @@
                         </div>
                     </b-col>
                 </b-row>
+                <b-row>
+                    <b-col>
+                        <b-row class="mb-2">
+                            <b-col>
+                                <strong
+                                    >December 17, 2020 - v1.3.1.4 - Immunization
+                                    Records
+                                </strong>
+                            </b-col>
+                        </b-row>
+                        <b-row>
+                            <b-col>
+                                <ul>
+                                    <li>
+                                        We launched the feature which will allow
+                                        Health Gateway users to view their
+                                        immunization records, including COVID-19
```

Comment:
@ninosamson can you review again the 2 terms "immunization records" vs "immunizations" for this new release notice?

Reply:
how do you mean?